scripts/Interface/ui/CMakeLists.txt

```diff
@@ -24,6 +25,7 @@ add_custom_target(CompileUIUI DEPENDS
   CompileUIInElastic
   CompileUIReflectometer
   CompileUISANS
+  CompileUIDataProcessorInterface
```

Comment:
I suppose I would consider what the main purpose of it is. Is it to provide an example of how to do this for users? If so then should we put it in to some kind of examples directory?